torch/csrc/TypeInfo.cpp

```diff
@@ -84,7 +84,7 @@ PyObject* THPIInfo_pynew(PyTypeObject* type, PyObject* args, PyObject* kwargs) {
   TORCH_CHECK(r.idx == 0, "Not a type");
 
   at::ScalarType scalar_type = r.scalartype(0);
-  if (!at::isIntegralType(scalar_type) && !at::isQIntType(scalar_type)) {
+  if (!at::isIntegralType(scalar_type, /*includeBool*/ true) && !at::isQIntType(scalar_type)) {
```

Comment:
```
print(torch.iinfo(np.bool)) --> iinfo(type=Bool)
print(torch.iinfo(torch.bool)) --> iinfo(type=Bool)
```

Reply:
I'm not sure what you are trying to show here.  Why should we return an iinfo that doesn't compute max?

Reply:
compute max? Im showing that it works same as for other types. For example, 

```print(torch.iinfo(torch.int64))``` prints ```iinfo(type=Long)```
```print(torch.iinfo(torch.uint8))``` prints ```iinfo(type=Byte)```
```print(torch.iinfo(np.int))``` prints ```iinfo(type=Long)```